parquet-hadoop/src/test/java/org/apache/parquet/hadoop/rewrite/ParquetRewriterTest.java

```diff
@@ -342,7 +377,12 @@ public void testPruneEncryptTranslateCodecSingleFile() throws Exception {
 
   @Test
   public void testPruneEncryptTranslateCodecTwoFiles() throws Exception {
-    testMultipleInputFilesSetup();
+    if (!inputFiles.contains(gzipEncryptionTestFileWithoutBloomFilterColumn)) {
```

Comment:
nit: perhaps we can wrap them in `ensureUncompressedFile()` and `ensureGzipFile()`?